spec/features/header_links_spec.rb

```diff
@@ -36,7 +36,7 @@
 
   context 'With non-MARC-based records' do
     before do
-      visit  '/catalog/dsp01ft848s955'
+      visit  '/catalog/dsp017s75dc44p'
```

Comment:
Question: Why is this changed?

Reply:
The dspace fixture I used was in my fixtures. The test is specific to dspace. The record page should not have cite, RefWorks, or EndNote links. https://catalog.princeton.edu/catalog/dsp01ft848s955 
https://catalog.princeton.edu/catalog/dsp017s75dc44p